File: samples/Java/Json Store/Cursor/Select/By Id.java
Store store = session.getDatabase("playground").getStore("pinball");
String s = ""; final StringBuilder b = new StringBuilder();

s += ">>> Select a document by ID\n";
Cursor c = store.openCursor().range(0,5);
b.setLength(0);

Document doc = store.loadDocument("1000");
int id = doc.getDocId();

c.id(id).find(new CursorHandler() {
  public boolean handle(CursorEntry e) throws JsonException {  
  	b.append("  value:"+e.getValue()+"\n");
    return true;
  }
});  
s += b.toString()+"\n";


_formatText("{0}",s);

File: samples/JavaScript/Json Store/Cursor/Select/By Id.js
session.setAsync(false);

var store = session.getDatabase("playground").getStore("pinball");
var s = "";

s += ">>> Select a document by Id\n";
var c = store.openCursor().range(0,5);
var doc = store.loadDocument("1000");
var id = doc.getDocId();

// Select by Id
c.id(id).find(function(e) {
  s += "  value:"+darwino.Utils.toJson(e.getValue(),false)+"\n";
});  
s += "\n";


darwino.Utils.setText("content","{0}",s);